platforms/software/testing-base-infrastructure/build.gradle.kts

```diff
@@ -0,0 +1,37 @@
+plugins {
+    id("gradlebuild.distribution.api-java")
+}
+
+gradlebuildJava.usedInWorkers()
+
+description = """Generalized test infrastructure to support executing tests in test workers."""
+
+dependencies {
+    api(projects.concurrent)
+    api(projects.javaLanguageExtensions)
+    api(projects.serialization)
+    api(projects.time)
+    api(project(":base-services"))
+    api(project(":logging"))
+    api(project(":logging-api"))
+    api(project(":worker-processes"))
```

Comment:
❌ We should pick a format (project accessors or stringy paths) and make it consistent in a single file.